transmart-batch/src/main/groovy/org/transmartproject/batch/facts/DummyFactGenerator.groovy

```diff
@@ -72,4 +110,30 @@ class DummyFactGenerator extends AbstractItemCountingItemStreamItemReader<Clinic
 
     @Override
     protected void doClose() throws Exception {}
+
+    private List<Map<String, Object>> getAssayIds(String sample_id) {
+        List result = jdbcTemplate.queryForList """
+                SELECT DISTINCT assay_id
+                FROM $Tables.SUBJ_SAMPLE_MAP
+                WHERE gpl_id = :gpl_id
+                AND sample_cd = :sample_cd
+            """, [gpl_id: platform.id, sample_cd: sample_id]
+        result
+    }
+
+    TrialVisit getTrialVisit() {
+        def trialVisit = jdbcTemplate.queryForList """
+                SELECT *
+                FROM $Tables.TRIAL_VISIT_DIMENSION
+                WHERE study_num = :study_num
+            """, [study_num: study.studyNum]
+
+        if (trialVisit) {
+            def t = trialVisit?.first()
```

Comment:
Would it work for the data people to bind row with a first trial visit from the study? If so you should do it once per job.

Reply:
I wasn't sure how to get a trial visit there and what to do, if there is more than one visit or none (for now there is a function to add a default visit then)